src/loaders/svg/tvgSvgPath.cpp

```diff
@@ -55,12 +55,22 @@ static bool _parseNumber(char** content, float* number)
 static bool _parseFlag(char** content, int* number)
 {
     char* end = NULL;
-    *number = strtol(*content, &end, 10);
-    //If the start of string is not number or a number was a float
-    if ((*content) == end || *end == '.') return false;
-    //If a flag has a different value than 0 or 1
-    if (*number != 0 && *number != 1) return false;
-    *content = _skipComma(end);
+    /* It handles the case what the flag of the arc of the path is defined
+    without spaces or commas, such as 00, 01, 11 or 10. */
+    if (*(*content) == '0' || *(*content) == '1') {
+        *number = *(*content) - '0';
+        *content += 1;
+        end = *content;
+        *content = _skipComma(end);
+    }
+    else {
+        *number = strtol(*content, &end, 10);
+        //If the start of string is not number or a number was a float
+        if ((*content) == end || *end == '.') return false;
+        //If a flag has a different value than 0 or 1
+        if (*number != 0 && *number != 1) return false;
+        *content = _skipComma(end);
+    }
```

Comment:
In my opinion if any of these two flags is something else than 0 or 1 it should be treated as an error.  I think that would be in agree with the specs. 
The thing is that in some online svg viewers (as w3school) it's not exactly like that. so the flags: ```0.0 1``` are considered to be wrong, but ```0 1.0``` are ok. So I'm a bit confused.

I think we should follow the specs, so I'd write this part like that:
```
static bool _parseFlag(char** content, int* number)
{
    if (*(*content) != '0' && *(*content) != '1') return false;
    *number = *(*content) - '0';
    *content = _skipComma(*content + 1);
    if (*(*content) == '.') return false;

    return true;
}
```
1. ```if (*(*content) != '0' && *(*content) != '1') return false;```  -> if there is something else than '0' or '1' false should be returned
2. ```if (*(*content) == '.') return false;```  -> if after the flag is a point it means that it was a float, so it's an error. In case it was some other NAN, it'll be handled in the next step, when the next value is loaded

What do you think? 
If you want to be in agree with w3school viewer (well, I don't know the algorithm they are using, I can check some cases to compare results), we can stay with you solution, but than I think we can remove the whole 'else' part and we don't really need the 'end' variable

Reply:
Thank you for your comment.
I'm a little concerned about corner case.
But I think the code you suggested is simpler and better.
I updated patch.